
<launch>



        <node pkg="eyes"  type="left_display.py" name="left_display" output="screen"></node>

        <node pkg="eyes"  type="right_display.py" name="right_display" output="screen"></node>

       <node pkg="eyes" type="face_detector.py" name="face_detector" output="screen"> </node>
        
        <node ns="camera" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
            <param name="approximate_sync" type="bool" value="true"/>
            <param name="queue_size" type="int" value="20"/>
            <param name="stereo_algorithm" type="int" value="200"/>

            <param name="prefilter_size" type="int" value="9"/>
            <param name="prefilter_cap" type="int" value="50"/>
    
             
            <param name="correlation_window_size" type="int" value="7"/>
            <param name="disparity_range" type="int" value="96"/>
            <param name="min_disparity" type="int" value="30"/>

            <param name="uniqueness_ratio" type="double" value="6.0"/>
            <param name="texture_threshold" type="int" value="10"/>
            <param name="speckle_size" type="int" value="1000"/>
            <param name="speckle_range" type="int" value="30"/>

            <param name="fullDP" type="bool" value="true"/>
            <param name="disp12MaxDiff" type="int" value="85"/>
            <param name="P1" type="double" value="0.0"/>
            <param name="P2" type="double" value="0.0"/>
            
        </node>

        <node pkg="eyes" type="depth_node.py" name="face_depth_pub" output="screen"> </node>
        
</launch>
